src/graphql/schema/uuid/abstract-repository/utils.ts

```diff
@@ -43,6 +44,29 @@ export function createRepositoryResolvers<
       if (!entity.currentRevisionId) return null
       return dataSources.serlo.getUuid<R>({ id: entity.currentRevisionId })
     },
+    async revisions(entity, cursorPayload, { dataSources }) {
+      const revsFromSerlo = await Promise.all(
+        entity.revisionIds.map((id) => {
+          return dataSources.serlo.getUuid<R>({ id })
+        })
+      )
+      const revs = revsFromSerlo.filter(isNotNil)
```

Comment:
no need for an extra variable:
```suggestion
      .filter(isNotNil)
```